pkgs/kcl-lang/kcl/pkg.yaml

```diff
@@ -1,32 +1,2 @@
 packages:
   - name: kcl-lang/kcl@v0.11.0
-  - name: kcl-lang/kcl
-    version: v0.10.8
-  - name: kcl-lang/kcl
-    version: v0.10.0-rc.1
-  - name: kcl-lang/kcl
-    version: v0.7.3
-  - name: kcl-lang/kcl
-    version: v0.5.1
-  - name: kcl-lang/kcl
-    version: v0.5.0
-  - name: kcl-lang/kcl
-    version: v0.4.6-beta.1
-  - name: kcl-lang/kcl
-    version: v0.4.6-alpha.2
-  - name: kcl-lang/kcl
-    version: v0.4.6-alpha.1
-  - name: kcl-lang/kcl
-    version: v0.4.5
-  - name: kcl-lang/kcl
-    version: v0.4.5-alpha.2
-  - name: kcl-lang/kcl
-    version: v0.4.4
-  - name: kcl-lang/kcl
-    version: v0.4.4-beta.2
-  - name: kcl-lang/kcl
-    version: v0.4.4-alpha.2
-  - name: kcl-lang/kcl
-    version: v0.4.4-alpha.1
-  - name: kcl-lang/kcl
-    version: v0.4.2-alpha.3
```

Comment:
We should support old versions as much as possible.
You should not remove old versions.
Removing alpha and beta is fine.

Reply:
I noticed many versions are missing as well from this list. Why is that?

Reply:
This is just test data.
You can install any version not listed here.